sw/device/tests/BUILD

```diff
@@ -384,6 +384,27 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
```

Comment:
this should use the new `opentitan_test` rule, not the old `opentitan_functest` rule as @moidx already changed this target over to the new build rules.

Reply:
it looks like you regressed to the old functest rule when moving this test to a new folder.

Reply:
That's right, and the whole tests/BUILD file uses opentitan_functest, so this should too.

Reply:
No, if you rebase on the latest master branch, you will see most tests do NOT use `opentitan_functest` anymore in this file, they use `opentitan_test`. You should NOT use `opentitan_functest` anymore.